scripts/upload_test_results/upload_to_github.py

```diff
@@ -0,0 +1,169 @@
+                upstream.git.pull()
+                print("Push updates to the origin ...")
+                origin.git.push('origin', 'master')
+
+                try:
+                    # Create a new branch for daily master results
+                    new_branch_name = branch_git_title + argument.commit_name
+                    origin.create_head(new_branch_name)
+                    print("Checkout to the new branch", new_branch_name)
+                    origin.git.checkout(new_branch_name)
+                except:
+                    print("Branch for the current commit already existing, skip step")
+
+
+            git_dir_path = os.path.join(repo_dir, 'results', ver_folder_name)
+            print("Destination directory path to store file in Git", git_dir_path)
+
+            if not os.path.exists(git_dir_path):
+                os.mkdir(git_dir_path)
+                print("Path created", git_dir_path)
+            else:
+                print("Path already existing")
+
+
+            # Step 2. Modify if necessary file name parsing function according to your .xml files name
+            # In my case file name was like junit-reel_board__zephyr_ae4ac87a91.xml
+
+            # Loop below moves and renames files from daily test directory to the git repository
+            for j in glob.glob("{}/*__zephyr*.xml".format(file_path)):
+                file = os.path.basename((j))
+                print("\nFile name", file)
+                file_meta = file.split("__")
+                file_branch = file.split("_")[-1]
+                file_branch = file_branch.split(".")
+                platform = file_meta[0].replace("junit-", "")
+                print("Platform", platform)
+                new_file_name = platform + '.xml'
+                print("New file name", new_file_name)
+
+                # Return path to its initial status
+                source_file_path = argument.res_dir
+                source_file_path = os.path.join(source_file_path, file)
+                print("Source directory file path", source_file_path)
+                git_file_path = os.path.join(git_dir_path, new_file_name)
+                print("Destination directory path file to commit", git_file_path)
+                shutil.copy(source_file_path, git_file_path)
+                origin.index.add(git_file_path)
+
+            try:
+                print("\nCommit changes ...")
+                author = Actor(author_name, author_email)
+                origin.git.commit('-m', ver_folder_name + ": " + company_name + " Master Daily Test Report" + \
+                "\n\nZephyr Daily Test Report for the master branch" + \
+                "\n\nSigned-off-by: " + author_name + " " + author_email, author=author)
+                print("Push commit to the origin ...")
+                origin.git.push('origin', new_branch_name)
+                print("Success! Go to your GitHub and create PR to merge the new origin branch with the upstream")
```

Comment:
When pushing the PR github returns an http address that you just have to click to create the PR.
It would be easier that this link is printed directly here, so user just have to click rather than open the browser, go to github, find test_result repo ....

Reply:
How to do that? In GitPython I can't find that option.

Reply:
I did that using export environment variable, please read README

Reply:
I haven't find the info.
Though, it should be easy to provide the link, it is built this way:
` https://github.com/<upstream_repo>/test_results/pull/new/<branch_name>`
So I my case, for instance, this is:
`https://github.com/erwango/test_results/pull/new/zephyr-v2.3.0-1733-g9df168b535_stm32`

Reply:
I made it in the updated script version there. Just use export environmental variable and after it script will print all info git messages.

Reply:
ok, i'll have a check, txs